React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -580,6 +580,16 @@ - (void)invalidateLayer
     layer.borderColor = borderColor;
     CGColorRelease(borderColor);
     layer.cornerRadius = (CGFloat)borderMetrics.borderRadii.topLeft;
+    if (@available(iOS 13.0, *)) {
```

Comment:
Not sure about adding the switch case directly in there, should I extract this in a function ?

Reply:
I guess you could extract it to a function like this one https://github.com/facebook/react-native/blob/de72e4d62e62eb5f98b4e39f9e4e126d7a9237d8/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm#L519

Reply:
@janicduplessis done!